nat-lab/bin/core-api.py

```diff
@@ -5,18 +5,29 @@
 from dataclasses import asdict, dataclass
 from http.server import BaseHTTPRequestHandler, HTTPServer
 from itertools import count
-from pprint import pprint
-from typing import Optional
+from typing import Dict, Optional
 from uuid import uuid4
 
+MOCKED_DERP_SERVER = {
+    "region_code": "mock",
+    "name": "mock",
+    "hostname": "mock-01",
+    "ipv4": "1.1.1.1",
+    "relay_port": 1111,
+    "stun_port": 1111,
+    "stun_plaintext_port": 1111,
+    "public_key": "mock",
+    "weight": 1,
+}
```

Comment:
It is fake data. 
As I mentioned in the PR description:
#Some of the data is mocked and the actual configuration of the peers is not implemented. This will be taken care of with further PRs as redesigning the current mesh_api implementation is required.

This is only to allow basic nat-lab tests to pass in case we will have an emulator with tests running in nat-lab earlier than full implementation. I.e. we will be able to register the device and get the meshmap, but of course not the pings.
The main issue is that we cannot re-use the methods/ variables/ classes from nat-lab/tests according to nat-lab architecture (here core-api is in /bin folder).
With further PRs it will be addressed.

Reply:
Why not just return empty data list instead, that way we could simply ignore it. Splitting the PR makes perfect sense, but the change could be introduced alongside other PR. Of course if the PR is from another repo and it's not ready yet, that might cause a problem

Reply:
Excellent question. There is validation on esp32 side when getting meshmap: if derp_servers == 0 then Fail, otherwise parse derp ip, pub key, port etc.

Reply:
Using real, already existing container is a great idea :) makes reviewing a breeze as well. I am happy with how this was resolved.